spaceship/spec/tunes/app_analytics_spec.rb

```diff
@@ -74,6 +74,25 @@
         a['adamId'].include?('898536088')
       end
       expect(val['meetsThreshold']).to eq(true)
+
+      measure_installs = analytics.app_measure_interval(start_time, end_time, 'installs')
+      expect(measure_installs['size']).to eq(1)
+      val = measure_installs['results'].find do |a|
+        a['adamId'].include?('898536088')
+      end
+      expect(val['meetsThreshold']).to eq(true)
+    end
+
+    it "grabs live analytics split by view_by" do
+      start_time, end_time = app.analytics.time_last_7_days
+      TunesStubbing.itc_stub_analytics(start_time, end_time)
+      analytics = app.analytics
+      measure_installs_by_source = analytics.app_measure_interval(start_time, end_time, 'installs', 'source')
+      expect(measure_installs_by_source['size']).to eq(5)
+      val = measure_installs_by_source['results'].find do |a|
+        a['adamId'].include?('898536088')
+      end
+      expect(val['meetsThreshold']).to eq(true)
```

Comment:
I've added only a single test case and a single stub for passing `view_by` parameter as they all would be pretty similar. To make it more understandable/distinguishable, I've extracted it to a separate test case.

Let me know if this is enough or do you want me to make stubs using `view_by` for each measure.